src/DetailView/DetailEpics.js

```diff
@@ -80,16 +81,32 @@ export const fetchEpic = (action$, store, call = (fn, ...args) => fn(...args)) =
           if (relationProperties.length !== 0) {
             return Observable.zip(
               ...relationProperties
-                .map(({key, id}) => {
+                .reduce((result, {key, id}) => {
```

Comment:
It's can be a map too

Reply:
Right, my first idea required `reduce`, but after some changes I forgot to change it to `map`. Thanks.